### PR TITLE
Aligned ALP topics to latest templates

### PR DESCRIPTION
**Pull request by tbazant**

### CODE REVIEW

File: references/ansible-playbook-examples.xml [hunk 1]
```diff
@@ -15,9 +15,9 @@
  xmlns:xi="http://www.w3.org/2001/XInclude"
  xmlns:xlink="http://www.w3.org/1999/xlink"
  xmlns:trans="http://docbook.org/ns/transclusion">
-  <title>Examples of &ansible; playbooks</title>
   <info>
     <meta name="maintainer" content="tbazant@suse.com"/>
```
Comment (janajaeger) [inline references/ansible-playbook-examples.xml:19]: its:translate="no" missing

File: references/virt-scenario-interactive-commands.xml [hunk 2]
```diff
@@ -15,14 +15,16 @@
  xmlns:xi="http://www.w3.org/2001/XInclude"
  xmlns:xlink="http://www.w3.org/1999/xlink"
  xmlns:trans="http://docbook.org/ns/transclusion">
-  <title>&virt-scenario;: interactive commands</title>
   <info>
     <meta name="maintainer" content="tbazant@suse.com"/>
```
Comment (janajaeger) [inline references/virt-scenario-interactive-commands.xml:19]: its:translate="no" missing

**Review by janajaeger — COMMENTED** [review]
just add a few its:translate attributes and you should be perfectly fine ;)

File: glues/alp-deployment-more-info.xml [hunk 3]
```diff
@@ -11,8 +11,8 @@
   xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink"
   xmlns:trans="http://docbook.org/ns/transclusion">
-  <title>For more information</title>
   <info>
+    <title>For more information</title>
     <meta name="maintainer" content="tbazant@suse.com"/>
```
Comment (tbazant) [inline glues/alp-deployment-more-info.xml:16]:
```suggestion
    <meta name="maintainer" content="tbazant@suse.com" its:translate="no"/>
```

File: glues/alp-workloads-more-info.xml [hunk 4]
```diff
@@ -11,9 +11,9 @@
   xmlns:xi="http://www.w3.org/2001/XInclude"
   xmlns:xlink="http://www.w3.org/1999/xlink"
   xmlns:trans="http://docbook.org/ns/transclusion">
-  <title>For more information</title>
   <info>
     <meta name="maintainer" content="tbazant@suse.com"/>
```
Comment (tbazant) [inline glues/alp-workloads-more-info.xml:15]:
```suggestion
    <meta name="maintainer" content="tbazant@suse.com" its:translate="no"/>
```

File: references/ansible-playbook-examples.xml (hunk 1 above)
Comment (tbazant) [inline references/ansible-playbook-examples.xml:19]:
```suggestion
    <meta name="maintainer" content="tbazant@suse.com" its:translate="no"/>
```

File: references/virt-scenario-interactive-commands.xml (hunk 2 above)
Comment (tbazant) [inline references/virt-scenario-interactive-commands.xml:19]:
```suggestion
    <meta name="maintainer" content="tbazant@suse.com" its:translate="no"/>
```

**Review by tbazant — COMMENTED** [review]
Added suggested its:translate="no" attributes

**Pull request merged**